### DISCUSSION TITLE
How to support Renovate?

### DISCUSSION THREAD

**Discussion by williamboman** [renovatebot/renovate#21938] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Hello! I'd like to support Renovate in any way I can. It's a very crucial component of many projects, including mine, and provides great value. Have you considered enabling people to sponsor the organization on GitHub, or are there any other venues where people can support Renovate? (I couldn't find any other discussions regarding this so apologies if this has been covered already)

**Comment by rarkins** [comment]
Hi @williamboman! Thank you for the nice post.

You are right that there's not really any way to support Renovate directly today except through Pull Requests, but I have been thinking about the topic.

Indirectly, we hope you will consider https://mend.io if you have needs for SCA (Open Source Vulnerability and License management) or SAST (Static analysis of first-party code).

The number 1 thing I'd like to achieve is the ability for companies in particular to be able to "sponsor" specific features they are interested in but without time or capabilities to implement. i.e. things which are a good idea to add, but which the core team simply doesn't have the time to contribute. This would ideally encourage an even wider community contribution if developers can be compensated for adding features which others want.

I might use your suggestion as motivation to look into GitHub Sponsors again, perhaps combined with something like OpenCollective.
